src/main/java/com/cmtelecom/text/sdk/models/templates/LocalizableParam.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2020 soufyan.bargach.
```

Comment:
Going to be super nit-picky here.
Can you set the Copyright to CM.com?

Not sure what Java-standards are, but normally a LICENSE file in the root specifies this, so you can leave this all out of each file.

Reply:
ah this is something that's get generated when creating a new class will delete those comments

Reply:
i have removed those comments ⚡️